src/org/intellij/erlang/completion/QuoteInsertHandler.java

```diff
@@ -65,13 +71,24 @@ public static void process(@NotNull Project project, @NotNull String name, @Null
       context.commitDocument();
       int tailOffset = context.getTailOffset();
       int startOffset = moduleName == null ? context.getStartOffset() : tailOffset - name.length();
-      document.insertString(startOffset, QUOTA);
-      document.insertString(tailOffset + 1, QUOTA);
+
+      if (!alreadyHasQuote(context, startOffset - 1)) {
+        document.insertString(startOffset, QUOTA);
+      }
+      if (!alreadyHasQuote(context, tailOffset)) {
+        document.insertString(tailOffset + 1, QUOTA);
+      }
```

Comment:
I don't like this code. we have the same code written two times. Please, do something about it.